package.json

```diff
@@ -21,7 +21,7 @@
     "@types/react-dom": "18.0.11",
     "eslint": "8.36.0",
     "eslint-config-next": "13.2.4",
-    "next": "13.2.4",
+    "next": "^13.2.4",
```

Comment:
Why do we need this change?

Reply:
Extremely sorry. Maybe this got added when running some dependencies.

Reply:
Should I remove this and add a new PR ?
Or will this one work ?

Reply:
Let's remove it from this PR since this PR is not addressing this change.

Reply:
I have removed that from the PR.
Please Check now.